tests/test_pure_pursuit.py

```diff
@@ -1,4 +1,8 @@
 import utilities.pure_pursuit as pp
+from utilities.pure_pursuit import PurePursuit
```

Comment:
This is already accessible as `pp.PurePursuit`; why import this into your own namespace?